community/settings.py

```diff
@@ -65,11 +66,16 @@
             'formatter': 'standard',
             'filters': ['noDebugFilter'],
             'level': 'INFO'
-        }
+        },
+        'log_to_stderr': {
+            'level': 'WARNING',
+            'class': 'logging.StreamHandler',
+            'stream': sys.stderr,
```

Comment:
Seems `stderr` is by default?

Reply:
Ya, probably, but I think it helps to be explicit here.

I havent 'closed' the associated issue, as this is a sub-optimal stop-gap solution.